activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -714,6 +715,16 @@ def extract_precision(sql_type)
       EMULATE_BOOLEANS_TRUE = { emulate_booleans: true }.freeze
 
       private
+        def strip_whitespace_characters(expression)
+          # These statements use single quotes because the expression may contain
+          # strings of these characters, _not_ the escaped versions that using
+          # double quotes would produce.
+          expression = expression.gsub('\n', "")
+          expression = expression.gsub('\\', "")
```

Comment:
It's unclear to me if you are trying to strip `\` or `\\`.

Also it would make sense to merge these two gsub together as a regexp, that would also fix the rubocop issue.

Reply:
👍🏻 I'll tweak that next.

Reply:
Done 😁 Thanks for the detailed feedback on this.